src/ast/rewriter/bv_rewriter.cpp

```diff
@@ -2803,7 +2803,26 @@ br_status bv_rewriter::mk_ite_core(expr * c, expr * t, expr * e, expr_ref & resu
     return BR_FAILED;
 }
 
-br_status bv_rewriter::mk_bvsmul_no_overflow(unsigned num, expr * const * args, bool is_overflow, expr_ref & result) {
+br_status bv_rewriter::mk_distinct(unsigned num_args, expr * const * args, expr_ref & result) {
+    if (num_args == 0) {
+        return BR_FAILED;
```

Comment:
result = true

Reply:
and if num_args = 1 then result = false.
OTOH, these rewrites are already handled by bool_rewriter

Reply:
Do you mean result = true for num_args = 1?

Reply:
yes, right, true for both 0 and 1 arguments.